cli/formatter/formatter.go

```diff
@@ -0,0 +1,62 @@
+package formatter
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/alibaba/pouch/apis/types"
+)
+
+// Format key to output
+const (
+	TableFormat = "table"
+	RawFormat   = "raw"
+)
+
+// IsTable to verify if table or raw
+func IsTable(format string) bool {
+	return strings.HasPrefix(format, TableFormat)
+}
+
+// PreFormat is to format the format option
+func PreFormat(format string) string {
+	if IsTable(format) {
+		format = format[len(TableFormat):]
+		// cut the space
+		format = strings.Trim(format, " ")
+		// input by cmd of "\t" is "\\t"
+		replace := strings.NewReplacer(`\t`, "\t", `\n`, "\n")
+		format = replace.Replace(format)
+
+		if format[len(format)-1:] != "\n" {
+			format += "\n"
+		}
+	} else {
+		format = "Name:{{.Name}}\nID:{{.ID}}\nStatus:{{.Status}}\nCreated:{{.Created}}\nImage:{{.Image}}\nRuntime:{{.Runtime}}\n\n"
+	}
+	return format
+}
+
+// LabelsToString is to transform the labels from map to string
+func LabelsToString(labels map[string]string) string {
+	var labelstring string
+	sortedkeys := make([]string, 0)
+	for key := range labels {
+		sortedkeys = append(sortedkeys, key)
+	}
+	sort.Strings(sortedkeys)
+	for _, key := range sortedkeys {
+		labelstring += fmt.Sprintf("%s = %s;", key, labels[key])
+	}
+	return labelstring
+}
+
+// MountPointToString is to transform the MountPoint from array to string
+func MountPointToString(mount []types.MountPoint) string {
```

Comment:
could you add a unit test for this func?

Reply:
done